src/renderers/dom/shared/DOMProperty.js

```diff
@@ -208,22 +212,34 @@ var DOMProperty = {
   getPossibleStandardName: __DEV__ ? {autofocus: 'autoFocus'} : null,
 
   /**
-   * All of the isCustomAttribute() functions that have been injected.
+   * Checks whether a property name is a writeable attribute.
+   * @method
    */
-  _isCustomAttributeFunctions: [],
+  isWriteable: function(name, value) {
+    if (DOMProperty.isReservedProp(name)) {
+      return false;
+    }
+
+    if (value == null || DOMProperty.properties.hasOwnProperty(name)) {
```

Comment:
What. _What_. 

The learning never ends. Do you know where I can learn more about why this checks `document.all`?